plugin/db/driver_test.go

```diff
@@ -208,3 +209,44 @@ func TestParseMigrationInfo(t *testing.T) {
 		})
 	}
 }
+
+func TestIsDoubleTimesAsteriskInTemplateValid(t *testing.T) {
+	tests := []struct {
+		template string
+		err      bool
+	}{
+		{
+			template: "**",
+			err:      true,
+		},
+		{
+			template: "bytebase/{{ENV_NAME}}/**",
+			err:      true,
+		},
+		{
+			template: "**/{{ENV_NAME}}/{{DB_NAME}}.sql",
+			err:      true,
+		},
+		{
+			template: "bytebase/**/{{ENV_NAME}}/**/{{DB_NAME}}__{{VERSION}}__{{TYPE}}__{{DESCRIPTION}}.sql",
+			err:      false,
+		},
+		{
+			template: "/**/{{ENV_NAME}}/**/{{DB_NAME}}__{{VERSION}}__{{TYPE}}__{{DESCRIPTION}}.sql",
+			err:      false,
+		},
+		// Credit to Linear Issue #1267
```

Comment:
nit: paste an actual link here

Reply:
SGTM. How about BYT-1267? I'm not sure if it's safe to expose connections used by Bytebase members.

Reply:
>I'm not sure if it's safe to expose connections used by Bytebase members.

Linear even [posts the entire issue body to the PR](https://github.com/bytebase/bytebase/pull/2389#issuecomment-1225697285) (if you link up PR and the Linear issue)... I don't think their product manager believes this is a concern 😂 but BYT-1267 works for me.